core/src/main/resources/defaults/blueprints/7.2.8/cdp-data-engineering.bp

```diff
@@ -226,7 +226,13 @@
           {
             "refName": "tez-GATEWAY-BASE",
             "roleType": "GATEWAY",
-            "base": true
+            "base": true,
+            "configs": [
+              {
+                "name": "tez-conf/tez-site.xml_client_config_safety_valve",
+                "value": "<property><name>tez.runtime.pipelined.sorter.lazy-allocate.memory</name><value>true</value></property>"
+              }
+                       ]
```

Comment:
This should be indented at the same level as the left end of `"configs": [`. To be changed in all the files.